mail.py

```diff
@@ -7,6 +7,7 @@
 import time
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+import argparse
```

Comment:
Can you move this to the top of the import list? It should be alphabetized within the block of the ones that start with `import ...`